wfl/Profiles/RestaurantProfile.cs

```diff
@@ -35,11 +35,28 @@ protected override void Configure(EntityConfiguration<Restaurant> configuration)
                     .WithLink<Restaurant, FoodOrdersController>(c => c.CreateFoodOrderAsync(null))
                     .WithField(x => x
                         .WithName(nameof(FoodOrderDto.RestaurantId))
-                        .WithType("text")
+                        .WithType("hidden")
                         .WithTitle("Restaurant Id")
                         .WithValue(r => r.Id)
                     )
                )
+               .UseActionTransform(actions => actions
+                    .WithName("edit-restaurant")
```

Comment:
One thing to think about from a long term API standpoint.  What is the significance of calling this action `edit-restaurant` instead of simply `edit`?  Is there additional information that a consumer of the API needs to know?  For API design it can be very helpful to keep action names consistent across entities which then allows consumers (including programatic consumers) to understand what actions mean across multiple entities.  

I don't think this makes a significant difference here, but I do think it's something interesting to keep in mind while thinking through API design.